### ISSUE TITLE
NVENC

### ISSUE THREAD

**Issue by PTT31** [dartraiden/NVIDIA-patcher#8]
patch driver support nvenv for mining card ?

**Comment by dartraiden** [comment]
Nope. Looks like NVENC support is disabled inside the chip (similar to PСI-E 3.0 support).

**Comment by dartraiden** [comment]
Some people report that NVENC is working on CMP XX cards after the [patch](https://github.com/keylase/nvidia-patch/tree/master/win).

**Comment by dartraiden** [comment]
English: Uploaded the latest release with the patch (NVENC- prefixed package).
